.github/workflows/ci.yml

```
@@ -11,21 +11,60 @@ on:
 jobs:
   ci:
     runs-on: ${{ matrix.os }}
-    container: docker://altendky/hydra:ubuntu-20.04-3-minors
     strategy:
       fail-fast: false
       matrix:
         os:
           - ubuntu-latest
+        python:
+          - version: "3.7"
+            bin: python3.7
+            tox: py37,report,codecov
+          - version: "3.8"
+            bin: python3.8
+            tox: py38,report,codecov
+          - version: "3.9"
+            bin: python3.9
+            tox: py39,report,codecov
+          - version: "3.10"
+            bin: python3.10
+            tox: py310,report,codecov
+          - version: "3.11"
+            bin: python3.11
+            tox: py311,report,codecov
+          - version: pypy-3.7
+            bin: pypy3
+            tox: pypy3,report,codecov
+    env:
```

If I'm reading correctly, I think we could do a single matrix still.

```suggestion
        include:
          - os: ubuntu-latest
            python:
              - version: "3.10"
                bin: python3.10
                tox: doc
          - os: ubuntu-latest
            python:
              - version: "3.10"
                bin: python3.10
                tox: lint
    env:
```

---

I kept it this way because it was like that before, but `os` doesn't really to be in the `matrix` since it's a single constant value.

---

Yeah, I guess I overlooked that change when making the example, though adjusting for it is minimal.  I often make "pointless" matrices and matrix axes because they make for less modification when copy/pasting snippets, but that doesn't seem to apply here.  I am somewhat tempted to test on macOS and Windows as well.  Anyways, we can go with this for now and maybe I'll try a follow up PR to compare.